### PR TITLE
Client-side_javascript_frameworks: fixed typos

### PR DESCRIPTION
**Pull request by OnkarRuikar**
## Summary
Fixed typos. Rephrased statements to correct the meaning or to simplify them.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md [hunk 1]
```diff
@@ -187,13 +187,13 @@ The vanilla JavaScript approach to building out new DOM elements in repetition w
 
 That's it. This snippet reduces approximately thirty-two lines of code down to six lines. If the curly braces and `v-` attributes here are unfamiliar to you, that's okay; you'll learn about Vue-specific syntax later on in the module. The thing to take away here is that this code looks like the UI it represents, whereas the vanilla JavaScript code does not.
 
-Thanks to Vue, we didn't have to write our own functions for building the UI; the framework will handle that for us in an optimized, efficient way. Our only role here was to describe to Vue what each item should look like. Developers who are familiar with Vue can join our project and quickly work out what is going on. Vue is not alone in this: using a framework improves team as well as individual efficiency.
+Thanks to Vue, we didn't have to write our own functions for building the UI; the framework will handle that for us in an optimized, efficient way. Our only role here was to describe to Vue what each item should look like. Developers who are familiar with Vue can quickly work out what is going on, when they join our project. Vue is not alone in this: using a framework improves team as well as individual efficiency.
```
Comment (dipikabh) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:190]:
```suggestion
Thanks to Vue, we didn't have to write our own functions for building the UI; the framework will handle that for us in an optimized, efficient way. Our only role here was to describe to Vue what each item should look like. Developers who are familiar with Vue can quickly work out what is going on when they join our project. Vue is not alone in this: using a framework improves team as well as individual efficiency.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
The changes look good, @OnkarRuikar! Just a comma needs to be dropped.

**Comment by OnkarRuikar** [comment]
@dipikabh done!

**Review by dipikabh — APPROVED** [review]

**Comment by dipikabh** [comment]
Thank you! 👍

**Pull request merged**